ballista-examples/Cargo.toml

```diff
@@ -0,0 +1,39 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+[package]
+name = "ballista-examples"
+description = "Ballista usage examples"
+version = "0.5.0-SNAPSHOT"
+homepage = "https://github.com/apache/arrow-datafusion"
+repository = "https://github.com/apache/arrow-datafusion"
+authors = ["Apache Arrow <dev@arrow.apache.org>"]
+license = "Apache-2.0"
+keywords = [ "arrow", "distributed", "query", "sql" ]
+edition = "2018"
+publish = false
+
+
```

Comment:
I wonder if it is worth adding `bin` targets here?

As it is I can't run these examples:

```shell
(arrow_dev) alamb@MacBook-Pro:~/Software/arrow-datafusion/ballista-examples$ cargo run 
error: a bin target must be available for `cargo run`
```

Maybe something like
```toml
[[bin]]
name = "dataframe"
path = "src/ballista_dataframe.rs"
```

Reply:
I was following the same pattern that we use in `datafusion-examples` where we use `cargo run --example` rather thaan `cargo run --bin`.

```
% cargo run --example
error: "--example" takes one argument.
Available examples:
    ballista-dataframe
    ballista-sql
```

It is a little odd that we package the examples in their own crate, so maybe packaging them as binaries makes more sense now?

Reply:
The main motivation from extracting them into a separate folder/crate for datafusion-examples was to reduce the nr of dependencies and compilation time.
Maybe bin works just as well?

Reply:
I have updated this to use `--bin` now.

Reply:
🎉